src/fragments/lib/push-notifications/react-native/interact_with_notifications/notification-opened.mdx

```diff
@@ -0,0 +1,43 @@
+## Respond to a notification being opened
+
+When a user taps on a notification displayed on their device, your app will be either launched or brought to the foreground. Knowing the contents of the notification the user interacted with can help you take further action (e.g. follow a deep link) or glean additional insight into your user engagement.
+
+To help you with this, Amplify provides two ways of handling notifications being opened. It is recommended that you handle both to ensure your users the most consistent and seamless experience.
+
+| App state  |       Handle with       |
+| :--------: | :---------------------: |
+| Foreground | `onNotificationOpened`  |
```

Comment:
maybe combine in the same row the ones similar? same feedback applies to the other tables.